<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget</class>
 <widget class="QTabWidget" name="TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Settings</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="DisplaySetttings">
   <attribute name="title">
    <string>Display Settings</string>
   </attribute>
   <widget class="QGroupBox" name="displayGroup_Subviews">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>371</width>
      <height>79</height>
     </rect>
    </property>
    <property name="title">
     <string>Subviews</string>
    </property>
    <widget class="QCheckBox" name="cbStatusBar">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>21</y>
       <width>102</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Status Bar</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbReuseSubView">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>49</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Reuse Subviews</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="displayGroup_Window">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Window Size</string>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="displayGroup_WindowSize">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QSpinBox" name="sbWidth">
          <property name="maximumSize">
           <size>
            <width>186</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sbHeight">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <widget class="QComboBox" name="cmbStdWinSize">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="cbOverride">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>29</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Render System Settings from OpenMW</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="displayGroup_Render">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>125</height>
     </rect>
    </property>
    <property name="title">
     <string>Render System</string>
    </property>
    <widget class="QComboBox" name="cmbRenderSys">
     <property name="geometry">
      <rect>
       <x>172</x>
       <y>42</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labAntiAlias">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>94</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Antialiasing</string>
     </property>
    </widget>
    <widget class="QLabel" name="labShaderLang">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>68</y>
       <width>84</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Shader Language</string>
     </property>
    </widget>
    <widget class="QLabel" name="labRenderSys">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>42</y>
       <width>104</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rendering Subsystem</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbShaderLang">
     <property name="geometry">
      <rect>
       <x>172</x>
       <y>68</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbAntiAlias">
     <property name="geometry">
      <rect>
       <x>172</x>
       <y>94</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>191</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_MaxSubviews">
       <property name="text">
        <string>Max Number of Subviews:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="sbMaxSubViews">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_MinSubviewWidth">
       <property name="text">
        <string>Min Subview Width:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="sbMinSubViewWidth">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>20000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>displayGroup_Render</zorder>
   <zorder>displayGroup_Subviews</zorder>
   <zorder>displayGroup_Window</zorder>
   <zorder>cbOverride</zorder>
   <zorder>layoutWidget_2</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbOverride</tabstop>
  <tabstop>cmbRenderSys</tabstop>
  <tabstop>cmbShaderLang</tabstop>
  <tabstop>cmbAntiAlias</tabstop>
  <tabstop>sbWidth</tabstop>
  <tabstop>sbHeight</tabstop>
  <tabstop>sbMaxSubViews</tabstop>
  <tabstop>sbMinSubViewWidth</tabstop>
  <tabstop>cbStatusBar</tabstop>
  <tabstop>cbReuseSubView</tabstop>
  <tabstop>cmbStdWinSize</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
